pkgs/servers/pulseaudio/qpaeq.nix

```diff
@@ -0,0 +1,51 @@
+{ mkDerivation
+, makeDesktopItem
+, python3
+, fetchurl
+, lib
+, pulseaudio
+}:
+
+let
+  desktopItem = makeDesktopItem {
+    name = "qpaeq";
+    exec = "@out@/bin/qpaeq";
+    icon = "audio-volume-high";
+    desktopName = "qpaeq";
+    genericName = "Audio equalizer";
+    categories = "Music;Sound;";
+    startupNotify = "false";
+  };
+in
+mkDerivation rec {
+  pname = "qpaeq";
+  inherit (pulseaudio) version src;
+
+  buildInputs = [
+    (
+      (python3.withPackages(ps: with ps; [ pyqt5 dbus-python sip ]))
+      .override(args: { ignoreCollisions = true; })
```

Comment:
I do not like this. Where does the colision occur and can we fix it? Otherwise a comment would be in order.

Reply:
sipconfig.py

It's because pyqt5 does provide it, but without the metadata